functional/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id 'java'
+    id 'idea'
+    id 'maven'
+}
+
+group = 'uk.sky.cirrus'
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenCentral()
+}
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.7'
+}
```

Comment:
We've now removed the need for a separate module, so I think this comment has implicitly been addressed